web/react/stores/post_store.jsx

```diff
@@ -150,6 +150,20 @@ var PostStore = assign({}, EventEmitter.prototype, {
   getDraft: function(channel_id, user_id) {
     return BrowserStore.getItem("draft_" + channel_id + "_" + user_id);
   },
+  storeCommentDraft: function(user_id, parent_post_id, draft) {
+    return BrowserStore.setItem(["comment_draft",user_id,parent_post_id].join("_"),draft);
+  },
+  getCommentDraft: function(user_id, parent_post_id) {
+    return BrowserStore.getItem(["comment_draft",user_id,parent_post_id].join("_"));
+  },
+  storeCurrentCommentDraft: function(parent_post_id, draft) {
+    var user_id = UserStore.getCurrentId();
+    return BrowserStore.setItem(["comment_draft",user_id,parent_post_id].join("_"),draft);
+  },
+  getCurrentCommentDraft: function(parent_post_id) {
+    var user_id = UserStore.getCurrentId();
+    return BrowserStore.getItem(["comment_draft",user_id,parent_post_id].join("_"));
```

Comment:
Yes, good call. @nickago can you remove the user_id from the key for your new comment drafts, as well as for the original drafts? They used to be needed before we started using BrowserStore